tests/php/_inc/lib/mocks/class-simplepie.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Implements a basic interface of the SimplePie class in environments where it doesn't exist.
+ *
+ * @package jetpack
+ */
+
+if ( ! class_exists( 'SimplePie' ) ) {
```

Comment:
Didn't know you have to do this...

Reply:
Normally we wouldn't need to do this, but this is an exceptional case:
- The `setup_tracks_callback` explicitly expects a `SimplePie` param: https://github.com/Automattic/jetpack/blob/091aff3cdcfbd112f8322879bbf1f905842ad58a/_inc/lib/class-jetpack-podcast-helper.php#L188
- Mocked objects don't change the method signature, so our mock expects it too.
- Since the class is unavailable in the Jetpack unit testing environment, we need to define it so we can use in our tests.
- However, this code is synced to WP.com and the class is available in the unit testing environment used there, so to avoid a "already declared" error we need to double check the class doesn't exist.

That way, the built-in `SimplePie` class provided by WordPress will be used when available, otherwise our simple fallback implementing the basics will be used.